cmd/root.go

```diff
@@ -48,6 +48,20 @@ to quickly create a Cobra application.`,
 	},
 }
 
```

Comment:
Looks good! In the future, all commands will the moved into the root.go command file